### PR TITLE
feat: [FFM-4443]: user friendly log msgs

### PR DESCRIPTION
**Pull request by crystal-fong**
### Summary

When a patch service is logged in the audit trail, it currently reads as `addService` or `removeService`
This is updated to read `Service {name} added` or `Service {name} removed` to improve readability

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11159)

File: src/modules/75-cf/components/AuditLogs/__tests__/AuditLogsList.test.tsx [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+    useGetAuditByParamsMock.mockReturnValue({
+      loading: false,
+      data: mockAuditData,
+      refetch: jest.fn(),
+      error: null
+    } as any)
+
+    renderComponent()
+
+    mockAuditData.data.auditTrails.forEach(trail => {
```
Comment (knagurski) [inline src/modules/75-cf/components/AuditLogs/__tests__/AuditLogsList.test.tsx:99]: Very thorough!

Might not be a good fit in this case (please don't change, it's good as-is 👍🏻), but for repeating tests, consider using `test.each`

```typescript
  test.each([
    ['case1', 'output1'],
    ['case2', 'output2'],
    ['case3', 'output3']
  ])('it should do a thing with %s and output %s', async (input, output) => {
    // ...
  })
```

That'll run the same test 3 times and pass in the test data for each scenario. Can be super-handy sometimes. Just FYI

File: src/modules/75-cf/components/AuditLogs/__tests__/AuditLogsList.test.tsx [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+    })
+  })
+
+  test('it should render correct user-friendly messages', async () => {
+    useGetAuditByParamsMock.mockReturnValue({
+      loading: false,
+      data: mockAuditData,
+      refetch: jest.fn(),
+      error: null
+    } as any)
```
Comment (knagurski) [inline src/modules/75-cf/components/AuditLogs/__tests__/AuditLogsList.test.tsx:95]: Looks like you do this same chunk in most tests, might as well move it to a `beforeEach` and just leave the overrides

File: src/modules/75-cf/components/AuditLogs/__tests__/AuditLogsList.test.tsx [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+const renderComponent = (): RenderResult =>
+  render(
+    <TestWrapper>
+      <AuditLogsList startDate={startDate} endDate={endDate} flagData={mockFeature} objectType="FeatureActivation" />
+    </TestWrapper>
+  )
+
+describe('AuditLogsList', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
```
Comment (knagurski) [inline src/modules/75-cf/components/AuditLogs/__tests__/AuditLogsList.test.tsx:29]: Doesn't look like you're asserting against any mocks that live outside individual tests. No harm in this, but it won't do anything.

Comment (crystal-fong) [reply src/modules/75-cf/components/AuditLogs/__tests__/AuditLogsList.test.tsx:29]: I think it's muscle memory at this point

**Review by knagurski — APPROVED** [review]
Excellent tests! Nice work

**Comment by crystal-fong** [comment]
trigger cypress

**Comment by crystal-fong** [comment]
retrigger jest

**Comment by crystal-fong** [comment]
retrigger jest

**Comment by crystal-fong** [comment]
retrigger jest

**Review by upadhyay-an — APPROVED** [review]

**Review by knagurski — APPROVED** [review]
Damn! 🔥

**Pull request merged**